resources/odbc/cleanup_driver.reg

```diff
@@ -0,0 +1,14 @@
+Windows Registry Editor Version 5.00
+
+; 64-bit driver on 64-bit Windows clean-up
+[-HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBC.INI\ADF_Test]
+
+[HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBC.INI\ODBC Data Sources]
+"ADF_Test"=-
+
+[HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBCINST.INI]
+
+[-HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBCINST.INI\ADF_ODBC_DRIVER]
+
+[HKEY_LOCAL_MACHINE\SOFTWARE\ODBC\ODBCINST.INI\ODBC Drivers]
+"ADF_ODBC_DRIVER"=-
```

Comment:
If we want to update the default driver name, now is a good time, no?

Reply:
Discussed offline and decided to keep it as is.